frontend/src/metabase/visualizations/components/settings/ChartSettingFieldsPartition.jsx

```diff
@@ -364,45 +374,42 @@ class Column extends React.Component {
     const showOptionsPanel = expanded && !isDragging;
     return connectDropTarget(
       connectDragSource(
-        <div
-          className="mb1 bordered rounded"
-          style={{
-            padding: "12px 14px",
-            "box-shadow": `0 2px 3px ${lighten(colors["text-dark"], 1.5)}`,
-            "&:hover": {
-              "box-shadow": `0 2px 5px ${lighten(colors["text-dark"], 1.3)}`,
-              transition: "all 300ms linear",
-            },
-          }}
-        >
-          <div
+        <div>
```

Comment:
Is this wrapping `div` required?

Reply:
Without the wrapper, the component fails to render with the following message:

> Uncaught Error: Only native element nodes can now be passed to React DnD connectors.You can either wrap ChartSettingFieldsPartition__DragWrapper into a `<div>`, or turn it into a drag source or a drop target itself.